specification/fabric/Microsoft.Fabric.Management/models.tsp

```diff
@@ -0,0 +1,176 @@
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+
+  @doc("The current state of Microsoft Fabric resource. The state is to indicate more states outside of resource provisioning.")
+  @visibility("read")
+  state: State;
+
+  @doc("The capacity administration")
+  administration: CapacityAdministration;
+}
+
+@doc("Represents the SKU name and Azure pricing tier for Microsoft Fabric capacity resource.")
+model RpSku {
+  @doc("The name of the SKU level.")
+  name: string;
+
+  @doc("The name of the Azure pricing tier to which the SKU applies.")
+  tier: RpSkuTier;
+}
+
+@doc("The name of the Azure pricing tier to which the SKU applies.")
+union RpSkuTier {
+  string,
+
+  @doc("Fabric tier")
+  Fabric: "Fabric",
+}
+
+@doc("The administration properties of the Fabric capacity resource")
+model CapacityAdministration {
+  @doc("An array of administrator user identities.")
+  members: Array<string>;
+}
+
+@doc("The resource provisioning state.")
+union ProvisioningState {
+  ResourceProvisioningState,
+
+  /** Resource is deleting */
+  Deleting: "Deleting",
+
+  /** Resource is provisioning */
+  Provisioning: "Provisioning",
+
+  /** Resource is updating */
+  Updating: "Updating",
+
+  string,
+}
+
+/** The resource provisioning state. */
+union State {
+  ResourceProvisioningState,
+
+  /** Resource is active */
+  Active: "Active",
+
+  /** Resource is provisioning */
+  Provisioning: "Provisioning",
+
+  /** Resource is updating */
+  Updating: "Updating",
+
+  /** Resource is deleting */
+  Deleting: "Deleting",
+
+  /** Resource is suspending */
+  Suspending: "Suspending",
+
+  /** Resource is suspended */
+  Suspended: "Suspended",
+
+  /** Resource is pausing */
+  Pausing: "Pausing",
+
+  /** Resource is paused */
+  Paused: "Paused",
+
+  /** Resource is resuming */
+  Resuming: "Resuming",
+
+  /** Resource is scaling */
+  Scaling: "Scaling",
+
+  /** Resource is preparing */
+  Preparing: "Preparing",
+
+  string,
+}
+
+@doc("Move resources request payload.")
+model MoveResourcesRequest {
+  @doc("The target resource group where the resources will be moved.")
+  targetResourceGroup: string;
+
+  @doc("The list of resources to move.")
+  resources: Array<string>;
+}
+
+@doc("An object that represents enumerating SKUs for new resources.")
+@pagedResult
+model RpSkuEnumerationForNewResourceResult {
+  #suppress "@azure-tools/typespec-azure-resource-manager/missing-x-ms-identifiers" ""
+  @doc("The collection of available SKUs for new resources")
+  value: Array<RpSkuDetailsForNewResource>;
+
+  @doc("Url for the next page.  Null if no more pages available")
+  @nextLink
+  nextLink?: string;
+}
+
+@doc("The SKU details")
+model RpSkuDetailsForNewResource {
+  @doc("The resource type")
+  resourceType: string;
+
+  ...RpSkuProperties;
+}
+
+@doc("The SKU properties")
+model RpSkuProperties {
+  @doc("The SKU's name")
+  name: string;
+
+  @doc("The list of available locations for the SKU")
+  locations: Array<string>;
```

Comment:
Can you use armLocation instead of unrestricted string?

Reply:
what's armLocation?
The default TrackedResource model defines location as string. Could you please provide a link to the documentation of this armLocation? I think it should be consistent with the "location" property on the Capacity resource which is string as seen in the below image.
![image](https://github.com/Azure/azure-rest-api-specs/assets/127952241/54385edd-fd71-46ae-8767-36d4099cad58)

Reply:
Somehow my reply got lost from the last iteration. Apologies, I meant to say azureLocation, which is documented here: [https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/data-types#Azure.Core.azureLocation](https://azure.github.io/typespec-azure/docs/libraries/azure-core/reference/data-types#Azure.Core.azureLocation).

It's a string under the covers, but indicates to typescript/swagger that the string must be formatted as an azure location.